apps/paymaster-nft-mint/src/libraries/aa-sdk/createModularAccountClientWithPaymaster.ts

```diff
@@ -32,10 +33,38 @@ export const createModularAccountClientWithPaymaster = async ({
     signer: signer,
     chain: chain,
     transport: http(rpcUrl),
+    entryPoint: getVersion060EntryPoint(
```

Comment:
requires specifying until aa-sdk is updated to have the correct entrypoint / factory addresses for new chains